client/js/components/procedure/admin/DpAddOrganisationList.vue

```diff
@@ -179,8 +179,8 @@ export default {
 
     rowItems () {
       return Object.values(this.invitableToebItems).reduce((acc, item) => {
-        const locationContactId = item.relationships.locationContacts.data[0].id
-        const locationContact = this.getLocationContactById(locationContactId)
+        const locationContactId = item.relationships.locationContacts.data.length > 0 ? item.relationships.locationContacts.data[0].id : null
```

Comment:
Is `locationContacts` always defined?

Reply:
I added an additional check to be safe: [2f9be6a](https://github.com/demos-europe/demosplan-core/pull/3649/commits/2f9be6a36bae17e98ba62533279f2cea6315f8e3)